docs/src/hugo/static/img/gitter.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="7" height="14">
```

Comment:
I didn't end up using this.  I'd like it to be a glyph like what comes from font-awesome, but I don't know how that magic works.